app/controllers/users/service_provider_inactive_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Users
+  class ServiceProviderInactiveController < ApplicationController
+    include FullyAuthenticatable
+
+    def index
+      analytics.track_event(Analytics::SP_INACTIVE_VISIT)
```

Comment:
sorry I'm late to the party, but would we want to also store which SP it is in the analytics?

Reply:
The SP is included by default in the track_event method. So when there is an sp this is included in the logged event (for example): 

`"service_provider":"urn:gov:gsa:openidconnect:sp:sinatra"`

Reply:
ah right, thanks!